src/node.ts

```diff
@@ -79,6 +85,7 @@ class Node extends VEventTarget {
     if (name in this.outputs) throw Error('Output name already exists');
     const socket = new OutputSocket(this);
     this.outputs[name] = socket;
+    this.dispatchEvent(new VEvent('addOutput'));
```

Comment:
Without extra info the event handler will not have idea which output was added. Perhaps the output name should be added as extra data.